Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/fedora:31
+FROM docker.io/library/fedora:33
```

Comment:
Should we use the fedora from fedora rather then through Docker?

Reply:
Good call!  Avoiding Docker Hub in CI is desirable with the latest rate limits.